ansible/roles/ocp-workload-fuse-ignite/tasks/workload.yml

```diff
@@ -1,178 +1,159 @@
-  changed_when: fuse_create_imagestream.rc == 0
-
-- name: Create fuse Operator resources in {{ ocp_project }}
-  shell: oc apply -f {{ fuse_online_operator_resources }} -n {{ ocp_project }}
-
-- name: TODO-REMOVE-THIS-AFTER-FUSE-ONLINE-INCLUDES-THESE-ARGS Patch args on operator to include monitoring resources addon
-  shell: oc patch dc/syndesis-operator -p='{"spec":{"template":{"spec":{"containers":[{"name":"syndesis-operator","args":["-a","/conf/addons"]}]}}}}}' -n {{ ocp_project }}
-
-- template:
-    src: templates/syndesis-customresource.yml.j2
-    dest: /tmp/syndesis-customresource.yml
-
-- name: Create Syndesis custom resource in {{ ocp_project }}
-  shell: oc apply -f /tmp/syndesis-customresource.yml -n {{ ocp_project }}
-
-#- name: Verify Fuse deployment succeeded
-#  shell: oc get pods -n {{ ocp_project }} --selector="app=syndesis" -o jsonpath='{.items[*].status.containerStatuses[?(@.ready==true)].ready}' | wc -w
-#  register: fuse_verify_result
-#  until: fuse_verify_result.stdout.find("8") != -1
-#  retries: 50
-#  delay: 10
-#  changed_when: False
-
-# Allow any authenticated user to access fuse. Sleep 5 to allow time for the
-# new deployment to kick off.
-#- name: Remove OpenShift SAR configuration on Fuse Managed OAuth proxy
-#  shell: oc get dc syndesis-oauthproxy -o yaml -n {{ ocp_project }} | sed '/--openshift-sar/d' | oc apply -f - -n {{ ocp_project }}; sleep 5
-
-#- name: Verify Fuse deployment succeeded
-#  shell: oc get pods -n {{ ocp_project }} --selector="app=syndesis" -o jsonpath='{.items[*].status.containerStatuses[?(@.ready==true)].ready}' | wc -w
-#  register: fuse_verify_result
-#  until: fuse_verify_result.stdout.find("8") != -1
-#  retries: 50
-#  delay: 10
-#  changed_when: False
-
-#- name: "Check Fuse readiness"
-#  block:
-#    - name: "Verify Fuse is running"
-#      shell: oc get pods --selector="app=syndesis" -o jsonpath='{.items[*].status.containerStatuses[?(@.ready==true)].ready}' -n {{ ocp_project }} | wc -w
-#      register: result
-#      until: result.stdout == "8"
-#      retries: 50
-#      delay: 10
-#      changed_when: False
-
-#    - name: "Verify syndesis instance is provisioned"
-#      shell: oc get syndesis {{ fuse_cr_name }} -o template --template \{\{.status.phase\}\}  -n {{ ocp_project }}  |  grep  'Installed'
-#      register: result
-#      until: result.stdout
-#      retries: 50
-#      delay: 10
-#      changed_when: False
+    namespace: "{{ ocp_project }}"
+    name: "{{ ocp_project }}-core-resource-limits"
+    api_version: v1
+    kind: LimitRange
+
+- name: create limitrange in namespace
+  k8s:
+    state: present
+    namespace:  "{{ ocp_project }}"
+    definition: "{{ lookup('template', role_path ~ '/templates/limitrange.j2' ) | from_yaml }}"
+
+- name: create CRD for Fuse Online
+  k8s:
+    state: present
+    definition: "{{ lookup('file', role_path ~ '/files/syndesis-crd.yml' ) | from_yaml }}"
```

Comment:
As long as they're all files or templates you can simplify quite a bit by using a `loop`. Like here (templates example): https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles_ocp_workloads/ocp4_workload_nexus_operator/tasks/workload.yml#L10